src/main.cpp

```diff
@@ -99,6 +99,11 @@ int main() {
     }
   }
 
+#ifdef DEBUG_BUILD
+  // Give time for debugger to attach
+  chThdSleepMilliseconds(1000);
+#endif
+
```

Comment:
Never needed a debugger delay, but if it helps you, I'm fine with it. But would it make sense to move this further up, in case you want to debug something that happens earlier?

Reply:
I didn't integrated it :innocent:, I only made it DEBUG_BUILD dependent. See here: https://github.com/xtech/fw-openmower-v2/pull/13#discussion_r2210035473